spec/requests/physical_switches_spec.rb

```diff
@@ -0,0 +1,100 @@
+describe "Physical Switches API" do
+  context "GET /api/physical_switch" do
+    it "returns all physical_switch" do
+      physical_switch = FactoryGirl.create(:physical_switch)
+      api_basic_authorize('physical_switch_show_list')
+
+      get(api_physical_switches_url)
+
+      expected = {
+        "name"      => "physical_switches",
+        "resources" => [{"href" => api_physical_switch_url(nil, physical_switch)}]
+      }
+      expect(response).to have_http_status(:ok)
+      expect(response.parsed_body).to include(expected)
+    end
+  end
+
+  context "GET /api/physical_switch/:id" do
+    it "returns one physical_switch" do
```

Comment:
same changes to reflect physical_switches

Reply:
done